source/wp-content/themes/wporg-parent-2021/patterns/404-page.php

```diff
@@ -0,0 +1,25 @@
+<?php
+/**
+ * Title: 404 Page
+ * Slug: wporg-parent-2021/404-page
+ * Categories: wporg
+ */
+
+?>
+
+<!-- wp:group {"className":"site-content-container"} -->
+<div class="wp-block-group site-content-container">
+	<p class="error404__oops" aria-hidden="true">
+		<?php esc_html_e( 'Oops!', 'wporg' ); ?>
+	</p>
```

Comment:
This should be in a block wrapper… but maybe it should actually be added in via CSS? Currently, since it's plain text, it can be highlighted:

![Screen Shot 2022-11-17 at 16 15 10](https://user-images.githubusercontent.com/541093/202562079-93513abd-146f-429c-a8c3-7f5c1a7a751e.png)

Adding it in CSS would loose the ability to translate it, but that's okay for now.